src/main/java/com/icthh/xm/tmf/ms/offering/web/rest/OfferingCatalogDelegate.java

```diff
@@ -26,12 +24,8 @@
     @Override
     @PrivilegeDescription("Privilege to get a product offering category")
     @LogicExtensionPoint(value = "ListCategory", resolver = ProfileKeyResolver.class)
-    @PreAuthorize("hasPermission({'profile': @headerRequestExtractor.get('Profile')}, 'QUALIFICATION.PRODUCT.OFFERING.GET')")
-    public ResponseEntity<List<Category>> listCategory(String channel,
-                                                       String profile,
-                                                       String acceptLanguage,
-                                                       Integer offset,
-                                                       Integer limit) {
+    @PreAuthorize("hasPermission({'profile': @headerRequestExtractor.get('Profile')}, 'OFFERING.CATEGORY.GET')")
```

Comment:
I think OFFERING.CATEGORY.LIST sounds betters, cause you're not retrieve category by ID, you get list of categories

Reply:
+

Reply:
Agree